pkg/apis/apps.kubermatic/v1/application_installation.go

```diff
@@ -70,7 +79,7 @@ type NamespaceSpec struct {
 	// +kubebuilder:validation:Type=string
 	Name string `json:"name"`
 
-	// +kubebuilder:default:=true
+	// +kubebuilder:default=true
```

Comment:
is it a typo mistake?

Reply:
Ah no. This has to do with swagger. Basically the swagger generator does not allow the `:=` notation. I also found this weird, because why would swagger care about +kubebuilder annotations, but for some reason it parses these. 

I guess we have done the same in other places as well https://github.com/kubermatic/kubermatic/blob/master/pkg/apis/kubermatic/v1/cluster.go#L265

Reply:
I had the same issue and my workaround was to leave an empty line above the field, e.g https://github.com/kubermatic/kubermatic/blob/b5263cd80d3679d6f05243eaeaeec1bafb5fa945/pkg/apis/kubermatic/v1/datacenter.go#L723-L727

Reply:
ah that is good to know. This also makes the swagger better looking.

```diff
--- a/cmd/kubermatic-api/swagger.json
+++ b/cmd/kubermatic-api/swagger.json
@@ -27715,7 +27715,7 @@
           "x-go-name": "Annotations"
         },
         "create": {
-          "description": "+kubebuilder:default=true\nCreate defines whether the namespace should be created if it does not exist. Defaults to true",
+          "description": "Create defines whether the namespace should be created if it does not exist.",
           "type": "boolean",
           "x-go-name": "Create"
         },
```

Reply:
We should consider refactoring this for all of our endpoints (of course in a separate PR ;)